xarray/plot/plot.py

```diff
@@ -40,14 +44,29 @@ def _ensure_plottable(*args):
     axis by matplotlib.
     """
     numpy_types = [np.floating, np.integer, np.timedelta64, np.datetime64]
+    # hacky
     other_types = [datetime]
+    try:
+        import cftime
+        cftime_datetime = [cftime.datetime]
+        if not nc_time_axis_available:
+            raise ImportError(
```

Comment:
I recommend switching to something more along the lines of my suggestion, for two reasons:
1. As written, this `ImportError` will always be caught in the `except` statement and will not be raised.
2. Ultimately, we also do not want to raise an `ImportError` for nc-time-axis if the user is not actively trying to plot cftime dates (in my suggestion I only check for the presence of nc-time-axis if cftime dates are present in one of the arguments).

Admittedly I did not test my suggestion thoroughly, though.  Was there something that went wrong with it?